pages/windows/msiexec.md

```diff
@@ -0,0 +1,20 @@
+# msiexec
+
+> Install, update, repair, or uninstall Windows programs using MSI and MSP package files.
+> More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec>.
```

Comment:
We don't use the i18n param in Microsoft URLs, it's auto detected based on user's language config
```suggestion
> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/msiexec>.
```